### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by arynn-gupta**

### CODE REVIEW

File: entry/src/main/js/default/i18n/en-US.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "strings": {
+    "hello": "Hello",
```
Comment (archanaatr) [inline entry/src/main/js/default/i18n/en-US.json]: Unwanted strings to be removed

File: entry/src/ohosTest/js/default/i18n/en-US.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "strings": {
+    "hello": "Hello",
```
Comment (archanaatr) [inline entry/src/ohosTest/js/default/i18n/en-US.json]: Unwanted strings please remove

File: entry/src/main/java/com/cyb0rg/neumorphism_smart_watch/MainAbility.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.cyb0rg.neumorphism_smart_watch;
```
Comment (archanaatr) [inline entry/src/main/java/com/cyb0rg/neumorphism_smart_watch/MainAbility.java:15]: Add license details in new files

**Review by archanaatr — COMMENTED** [review]
Please check the comments.

**Comment by arynn-gupta** [comment]
1. Added License Header
2. Deleted Unwanted strings

File: entry/src/main/js/default/pages/index/index.hml [hunk 4]
```diff
@@ -0,0 +1,76 @@
+<div class="body">
```
Comment (archanaatr) [inline entry/src/main/js/default/pages/index/index.hml:18]: Request to use these components and update the application.
https://github.com/applibgroup/Theme
https://github.com/applibgroup/Alert
https://github.com/applibgroup/Buttons
https://github.com/applibgroup/Card
https://github.com/applibgroup/Checkbox
https://github.com/applibgroup/Dropdown
https://github.com/applibgroup/form
https://github.com/applibgroup/Navbar
https://github.com/applibgroup/pagination
https://github.com/applibgroup/progress

**Review by archanaatr — COMMENTED** [review]
Pls check the comment

**Review by archanaatr — APPROVED** [review]
Ok

**Pull request merged**